Sources/InterestsHash.swift

```diff
@@ -0,0 +1,12 @@
+import Foundation
+
+struct InterestsHash {
```

Comment:
Do we need a new struct for this? Is there an equivalent for the "DeviceStateStore" that we have in Android?

Seems a bit of an overkill and repetitive to do `InterestsHash().serverConfirmedInterestsHash()`.

Reply:
Indeed, we do. I thought about adding this functionality in [InterestPersistable](https://github.com/pusher/push-notifications-swift/blob/master/Sources/InterestPersistable.swift) but it didn't feel right to me.

Would you prefer having:
- `func persist(_ hash: String)`
- `func serverConfirmedInterestsHash() -> String`

in `InterestPersistable`?

Reply:
Yes, but with a more verbose: `func persistServerConfirmedInterestsHash(_ hash: String)`